h/schemas/forms/accounts/login.py

```diff
@@ -11,13 +11,17 @@
 @colander.deferred
 def _deferred_username_widget(node, kw):
     """Return a username widget that autofocuses if username isn't pre-filled."""
-    return deform.widget.TextInputWidget(autofocus=_should_autofocus_username(kw))
+    return deform.widget.TextInputWidget(
+        autofocus=_should_autofocus_username(kw), autocomplete="username"
```

Comment:
I think "username" is okay here even though its either username or email, we just have to pick one.

Reply:
See my comment above.